acceptance/suites/pre_suite/puppet4_compat/70_install_puppet.rb

```diff
@@ -0,0 +1,11 @@
+require 'puppetserver/acceptance/compat_utils'
+
+step "Install Legacy Puppet Agents."
+
+puppet_agent_version = '1.9.3'
```

Comment:
Do we want to have this hardcoded / use of the environment variable commented out below.  That would make the value not possible to override from ci-job-configs, right?  Not sure if that's a big deal but different than what we've done for this in the past anyway.